src/widgets/study-with-us/ui/study-with-us.tsx

```diff
@@ -23,14 +26,21 @@ export const StudyWithUs = () => (
             Currently 500+ developers from different countries and companies involve in the
             education process as mentors.
           </Paragraph>
-        </div>
-        <Image className="right picture" src={image} alt="education" />
+        </article>
+        <Image
+          className={cx('picture')}
+          src={rsSchool}
+          alt="Slot - mascot in glasses and works at a laptop"
+        />
       </div>
-      <div className="study-options column-3 ">
+      <div className={cx('study-options')}>
         {studyOptions.map((i) => (
-          <OptionItem {...i} key={i.title} />
+          <article key={i.title} className={cx('option-item')} data-testid="option-item">
+            <Subtitle>{i.title}</Subtitle>
+            <Paragraph fontSize="large">{i.description}</Paragraph>
+          </article>
```

Comment:
```suggestion
          <article key={studyOption.title} className={cx('option-item')} data-testid="option-item">
            <Subtitle>{studyOption.title}</Subtitle>
            <Paragraph fontSize="large">{studyOption.description}</Paragraph>
          </article>
```
Maybe we should improve the naming, "i" - this is iteration or index. If we iterate over options, maybe it is better to indicate that there is one option

Reply:
Done